Dockerfile

```diff
@@ -25,8 +25,10 @@ RUN mkdir /app && unzip /jenkinsfile-runner/vanilla-package/target/war/jenkins.w
 
 FROM openjdk:8-jdk
 ENV JENKINS_UC https://updates.jenkins.io
+ENV JENKINS_PM_VERSION 0.1-alpha-10
+ENV JENKINS_PM_URL https://github.com/jenkinsci/plugin-installation-manager-tool/releases/download/plugin-management-parent-pom-$JENKINS_PM_VERSION/jenkins-plugin-manager-$JENKINS_PM_VERSION.jar
```

Comment:
I wonder why it is located in POM instead of CLI. CC @stopalopa

Reply:
Do you mean that we should move these lines to be installed through the POM ?

Reply:
Sorry, missed the ping. No, no action required here. It is rather about the GitHub releases deployment